streaming/src/main/scala/io/gearpump/streaming/task/SerializerResolver.scala

```diff
@@ -0,0 +1,30 @@
+package io.gearpump.streaming.task
+
+import io.gearpump.esotericsoftware.kryo.util.ObjectMap
+import io.gearpump.esotericsoftware.kryo.util.IntMap
+import io.gearpump.streaming.task.SerializerResolver.Registration
+
+private[task] class SerializerResolver {
```

Comment:
Hi Weihua, generally this refactor is transparent to user. 
We have 5 type of messages will be transported by the netty layer, including ```Message Ack AckRequest InitialAckRequest LatencyProbe```. However, user's message is boxed in ```Message``` and it will still be serialized by Kryo, which means user only need to care about registering their serializers when running a streaming application, so the ```SerializerResolver``` is actually only be used in Gearpump internally.